Cloudinary.Test/CloudinaryTest.cs

```diff
@@ -705,11 +705,12 @@ public void TestPublishByTag()
                 ResourceType = ResourceType.Image
             });
 
-            Assert.AreEqual(publish_result.Published.Count, 1);
+            
 
             DelResResult delResult = m_cloudinary.DeleteResourcesByTag(
                 "TestForPublish");
 
+            Assert.AreEqual(publish_result.Published.Count, 1);

```

Comment:
why did you change the order? it's more clear when you check the count right after the operation.